src/components/Header/Header.jsx

```diff
@@ -1,13 +1,17 @@
-import Navbar from "../Navbar/Navbar";
+import { useContext } from "react";
+import Navbar from '../Navbar/Navbar';
+import { AuthContext } from "../../containers/AuthContext";
 // import styles from './Header.module.scss';
 
 export default function Header() {
-  return (
-    <header>
-      <div className="container">
-        <p>Header </p>
-        <Navbar />
-      </div>
-    </header>
-  );
+    const authContext = useContext(AuthContext);
```

Comment:
what do we do else with `authContext` except logging it on the console ? :)

Reply:
we fulfill Jira requirements #3 :)
![Uploading image.png…]()

Reply:
if it is no longer needed - I can delete it))

Reply:
fixed